packages/manager/src/constants/API_ENDPOINTS.ts

```diff
@@ -33,6 +34,7 @@ export const API_ENDPOINTS: APIEndpoints = (() => {
 					process.env.slice_machine_v1_endpoint ??
 						"https://mc5qopc07a.execute-api.us-east-1.amazonaws.com/v1/",
 				),
+				RepositoryService: "https://repository.internal.wroom.io/",
```

Comment:
⚠️ You should manage the environment variables setup as other endpoint here. It should come from the .env of the dev env directly here right? Is `RepositoryService` fully functional is the dev env already as it's part of Obelix?

Reply:
Changed to check `process.env` first and use this as a fallback ✅